### PR TITLE
Fix SickRage/sickrage-issues/issues/813

### PR DESCRIPTION
**Pull request by fernandog**
Fix https://github.com/SickRage/sickrage-issues/issues/813

User can choose to PP even if no subtitle is available even if setting "postpone if no subs" is enabled
and it wont postpone if show doesn't have subs enabled

@MGaetan89 updates webapi

**UPDATE:**
known issues:
this filename `[HorribleSubs] Boku dake ga Inai Machi - 07 [720p].mkv` raises an error in subliminal causing:

```
FINDSUBTITLES :: Error occurred when downloading subtitles for: /media/[HorribleSubs] Boku dake ga Inai Machi - 07 [720p].mkv. Error: Insufficient data to process the guess
```

when that happen user should manual PP with skipping subtitle check so it can be PP
But even after PP subliminal will still raise an error while searching for subs.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
I recall a time when I said this "dont pp if no subs" would result in a never ending line of patches and workarounds to make it work. I thought "Its done, it works perfect"?

File: contrib/nzbToMedia [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 3e4861e87c543a4d2debd8997feb16bda3ef5339
```
Comment (fernandog) [inline contrib/nzbToMedia]: why this got updated? I didn't change

File: contrib/nzbToMedia [hunk 2]
```diff
@@ -1 +1 @@
...
+Subproject commit 1cd64053b0da1d351bb6af392e1ebb904c6b6ce2
```
Comment (fernandog) [inline contrib/nzbToMedia]: why this got updated? I didn't change

**Comment by fernandog** [comment]
Its not a workaround. Its a setting in manual PP like failed, priority, re-process, etc
and the setting will only appear in PP if user has the feature enabled
this was intended to be done in the original PR. But did'nt want to do a lot of changes in one PR

**Comment by miigotu** [comment]
Except what about automatic pp... Show don't have subs enabled but postpone if no subs...

**Comment by fernandog** [comment]
@miigotu
~~Its not the scope of this PR. This fixes part of the issue.
Will do another PR for this other part~~

**Comment by miigotu** [comment]
You cant change params of existing methods in the webapi.

**Comment by fernandog** [comment]
~~It not a required field.
Not an issue~~

**Comment by miigotu** [comment]
it is an issue, because nzbtomedia works by matching the method signature. add/remove/rename a parameter and it stops working.

**Comment by fernandog** [comment]
~~as I said, it works. nzbtomedia calls the url without the parameter.
It took a while because I dont pay for news server. Use a free one.~~

https://gist.github.com/fernandog/b63d8daf72f8061e4c6e

**Comment by MGaetan89** [comment]
@fernandog I'm fine with the API change, as it is an optional new parameters.

**Comment by fernandog** [comment]
@miigotu can you review please?

**Comment by fernandog** [comment]
@felipesalomao

**Comment by fernandog** [comment]
@miigotu Fixed signature

**Comment by labrys** [comment]
If you modify a methods signature or add a new method you are responsible for making sure the docstring is up to date and has all params, _args, and *_kwargs documented.

**Comment by labrys** [comment]
Approved!

[Approved with PullApprove](https://pullapprove.com/SickRage/SickRage/pull-request/981/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=SickRage/SickRage)

**Pull request merged**